lib/Spreadsheet/ParseXLSX.pm

```diff
@@ -464,11 +455,17 @@ sub _parse_sheet {
 
     $sheet_xml->parse( $sheet_file );
 
-    for my $key (keys %merged_cells) {
-        $cells{$key}{Merged} = 1 if $cells{$key};
-    }
-
-    if ( ! $sheet->{Cells} ){
+    if ( $sheet->{Cells} ){
+      for (my $r = 0; $r < scalar(@{$sheet->{Cells}}); $r++) {
```

Comment:
Why a C-style for loop? This could be done more directly as
```perl
for my $r (0 .. $#{$sheet->{Cells}}) {
```

Reply:
Heh, can't deny my roots.